package/kernel/linux/modules/netdevices.mk

```diff
@@ -1344,7 +1344,8 @@ define KernelPackage/mlx5-core
 	CONFIG_MLX5_MPFS=y \
 	CONFIG_MLX5_SW_STEERING=n \
 	CONFIG_MLX5_TC_CT=n \
-	CONFIG_MLX5_TLS=n
+	CONFIG_MLX5_TLS=n \
+	CONFIG_MLX5_VFIO_PCI=n@ge5.18
```

Comment:
You can always set this, if `CONFIG_MLX5_VFIO_PCI` does not exist on older kernel versions it does nothing.

Reply:
Done. Thanks.